City-of-Helsinki/drupal-hdbt
# UHF-X accordion state fix

Accordion had problems saving local storage state when dealing with multiple different pages.

## What was done
- Saving local storage state works properly with multiple pages.
- Added button type button to accordion buttons (accessibility)

## How to install

[Check this out](https://github.com/City-of-Helsinki/drupal-hdbt/pull/541)

## How to test

Pretty much [same as this.](https://github.com/City-of-Helsinki/drupal-hdbt/pull/541)
- Set accordions on multiple pages and open and close them. The state should be saved without errors.
- Accordion buttons on accordion items should have `type="button"` -attribute
